src/App.js

```diff
@@ -1,5 +1,7 @@
 import React, { Component } from 'react';
 
+import School from './scenes/School';
```

Comment:
Since School is just a visualization and not an entire page, let's place it under components instead